packages/core/src/rum/instrumentation/resourceTracking/DdRumResourceTracking.tsx

```diff
@@ -37,7 +38,10 @@ export class DdRumResourceTracking {
             return;
         }
 
-        this.requestProxy = new XHRProxy(XMLHttpRequest);
+        this.requestProxy = new XHRProxy({
+            xhrType: XMLHttpRequest,
+            resourceReporter: new ResourceReporter([])
```

Comment:
here we'll be able to pass the user defined resource mapper :)

Reply:
The changes in the public API exposing this possibility will be in another PR? Just cannot see it in the current one.

Reply:
It will be part of another PR later. I realized we need to do some validation and set some error boundaries if we want to implement a user defined resource mapper.
Also we want to limit what the user can change, how many fields they are allowed to add and so on.